package.json

```diff
@@ -16,9 +18,23 @@
   },
   "homepage": "https://github.com/zhoutwo/url_connect#readme",
   "devDependencies": {
+    "babel": "^6.23.0",
```

Comment:
I thought we are using TypeScript? TypeScript can transpile JSX. Name it TSX. See [https://www.typescriptlang.org/docs/handbook/jsx.html](https://www.typescriptlang.org/docs/handbook/jsx.html).